src/lib/annotations/Annotator.js

```diff
@@ -212,12 +221,12 @@ class Annotator extends EventEmitter {
         }
 
         // Hide create annotations button if image is rotated
-        const annotateButton = this.previewUI.getAnnotateButton();
+        const pointAnnotateButton = this.previewUI.getAnnotateButton(SELECTOR_BOX_PREVIEW_BTN_ANNOTATE_POINT);
```

Comment:
The check is done in the annotatorUtil.hide and show element. A rotateAnnotation's test will need to be rewritten if we add in an extra check after previewUI.getAnnotateButton(selector).